app/ide-desktop/lib/client/src/projectManagement.ts

```diff
@@ -0,0 +1,237 @@
+/** @file This module contains functions for importing projects into the Project Manager.
```

Comment:
we already have `src/bin/project-manager`. Having also `src/projectManagement` (btw, wring name style), is confusing. Can we somehow merge them or rename this file?

Reply:
> we already have `src/bin/project-manager`. Having also `src/projectManagement` (btw, wring name style), is confusing. Can we somehow merge them or rename this file?

I'd recommend _not_ merging, as there is nice responsibility separation (Project Mana-_ger_ bindings vs Project Mana-_gement_)